src/openassetio-python/package/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -252,31 +252,70 @@ class Test_entityExists(FixtureAugmentedTestCase):
     managerApi.ManagerInterface.entityExists.
     """
 
-    def setUp(self):
-        self.a_reference_to_an_existing_entity = self._manager.createEntityReference(
-            self.requireFixture("a_reference_to_an_existing_entity", skipTestIfMissing=True)
+    def test_when_querying_existing_reference_then_true_is_returned(self):
+        a_reference_to_an_existing_entity = self.requireEntityReferenceFixture(
+            "a_reference_to_an_existing_entity", skipTestIfMissing=True
         )
-        self.a_reference_to_a_nonexisting_entity = self._manager.createEntityReference(
-            self.requireFixture("a_reference_to_a_nonexisting_entity")
+        context = self.createTestContext()
+        result = [None]
+        self._manager.entityExists(
+            [a_reference_to_an_existing_entity],
+            context,
+            lambda idx, value: operator.setitem(result, idx, value),
+            lambda idx, error: self.fail(f"entityExists should not fail: {error.message}"),
         )
+        self.assertEqual(result, [True])
 
-    def test_existing_reference_returns_true(self):
+    def test_when_querying_nonexisting_reference_then_false_is_returned(self):
+        a_reference_to_a_nonexisting_entity = self.requireEntityReferenceFixture(
+            "a_reference_to_a_nonexisting_entity", skipTestIfMissing=True
+        )
         context = self.createTestContext()
-        assert self._manager.entityExists([self.a_reference_to_an_existing_entity], context) == [
-            True
-        ]
+        result = [None]
+        self._manager.entityExists(
+            [a_reference_to_a_nonexisting_entity],
+            context,
+            lambda idx, value: operator.setitem(result, idx, value),
+            lambda idx, error: self.fail(f"entityExists should not fail: {error.message}"),
+        )
+        self.assertEqual(result, [False])
 
-    def test_non_existant_reference_returns_false(self):
+    def test_when_querying_existing_and_nonexisting_references_then_true_and_false_is_returned(
+        self,
+    ):
+        a_reference_to_an_existing_entity = self.requireEntityReferenceFixture(
+            "a_reference_to_an_existing_entity", skipTestIfMissing=True
+        )
+        a_reference_to_a_nonexisting_entity = self.requireEntityReferenceFixture(
+            "a_reference_to_a_nonexisting_entity", skipTestIfMissing=True
+        )
```

Comment:
[consider] In the old one, we forced the provision of a non-existing ref if you provided an existing one (to try and avoid parts of the check being skipped inadvertently).  Might be good to keep?

Reply:
:thinking: I do think we need to properly define groups of tests that should be run together, and probably change how that's configured (e.g. see  https://github.com/OpenAssetIO/OpenAssetIO/issues/1031)

Will change to behave as you describe for now.

Reply:
I'll push a quick fix to BAL to correspond.  Still don't like it, but that's a bigger discussion.

Reply:
Yeah, we need to re-think the whole apiCompliance thing dont we.